inst/private/check_and_convert.m

```diff
@@ -0,0 +1,38 @@
+function obj = check_and_convert()
+  newl = sprintf('\n');
+  
+  str_check_is_list = strjoin({'if isinstance(_outs, (list, tuple)):',
+                              '  is_list = True',
+                              'else:',
+                              '  is_list = False'},newl);
+  pyexec(str_check_is_list);
+  is_list = pyeval('is_list');
+  
+  if is_list
+    n = pyeval('len(_outs)');
+  else
+    n = 1;
+  end
+  
+  obj = [];
```

Comment:
cell array:
`obj = {}`.

If one output is a dict and the next is a sym, then they cannot go in an array together.